dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -336,3 +338,48 @@ def country_code_to_name(country_code):
         return record[code]['item'][0]['name']
 
     raise requests.exceptions.RequestException(register_response)
+
+
+def unsuspend_suspended_supplier_services(record, suspending_user, client, logger, dry_run):
+    """
+    This method checks if the supplier has services suspended by the suspending_user, if so it un-suspends them. It
+    returns None.
```

Comment:
Given that you've got `:return: None` below, do we need `It returns None` as well?